src/main/js/dynamic/loaded/audit/Details.js

```diff
@@ -157,6 +157,7 @@ class Details extends Component {
                                 </div>
                             </Tab>
                         </Tabs>
+                        <p name="message">{this.props.errorMessage}</p>
```

Comment:
Do we not want to use the StatusMessage here and leave the action message empty?

Reply:
That's the dialog for audit.  Status message is shown underneath the modal.  All modals right now show errors this way. 
So the modals are at least consistent.